Userland/Libraries/LibVideo/CodecID.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023, Stephan Vedder <stephan.vedder@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Format.h>
+
+namespace Video {
+
+enum class CodecID : u32 {
+    Unknown,
+    // On2 / Google
+    VP8,
+    VP9,
+    // MPEG
+    H261,
+    MPEG1,
+    H262,
+    MPEG2 = H262,
+    H263,
+    H264,
+    AVC = H264,
```

Comment:
I'm not sure if we benefit at all from aliasing these, it seems like that could just lead to confusion. Better to just have a consistent naming style and use it everywhere we can.

Reply:
Given that this is an internal enum, and it doesn't differentiate between these aliases, it doesn't do us any good to use multiple names. If we were to keep the naming style from the container, perhaps there's an argument for that, but then if we display the codec to the user eventually, they may not know that H.265 is the same as HEVC.

Anyone writing code to make use of these values should know what the different names mean. When displaying the codec to the user, we should either stick with one name, or give all aliases at once in the human-friendly text.